OPAL/br/src/main/scala/org/opalj/br/fpcf/properties/fieldaccess/package.scala

```diff
@@ -0,0 +1,42 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package br
+package fpcf
+package properties
+
+import org.opalj.br.analyses.DeclaredMethods
+import org.opalj.value.ValueInformation
+
+import scala.collection.immutable.IntMap
+
+package object fieldaccess {
+    // we encode field accesses (method, pc) as longs
+    // MSB 16 bit unused | 16 bit PC | 32 bit MethodID LSB
+    type FieldAccess = Long
+
+    type AccessReceiver = Option[(ValueInformation, PCs)]
+    type AccessParameter = Option[(ValueInformation, PCs)]
+
+    type IndirectAccessReceivers = IntMap[IntMap[AccessReceiver]] // Access Context => PC => Receiver
+    type IndirectAccessParameters = IntMap[IntMap[AccessParameter]] // Access Context => PC => Parameter
+
+    @inline def encodeDirectFieldAccess(method: DefinedMethod, pc: Int): FieldAccess = {
+        method.id.toLong | (pc.toLong << 31)
```

Comment:
Doesn't fit the above comment of using 32 bit for the MethodID. Probably a result of the wrong comment in pointsto/package.scala that wasn't updated to reflect that that one now uses 17 bit for PC and 26 bit for the ContextID.